### PR TITLE
Increase memory requests and limits

### PR DESCRIPTION
**Pull request by bipuladh**

### CODE REVIEW

**Comment by SanjalKatiyar** [comment]
@bipuladh plz rebase with latest master, commit: 00669b9a8c237e0e46bb5ad63cb85c6ed778d4dd is already merged !

File: ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml [hunk 1]
```diff
@@ -25,10 +25,15 @@ releases:
 resources:
   '*':
     limits:
-      memory: 4Gi
+      memory: 12Gi
     requests:
       cpu: 100m
       memory: 200Mi
```
Comment (alfonsomthd) [inline ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml:31]: @bipuladh Can we increasee these generic soft limits ? Tha should speed up the time spent by the plugin building container.

**Comment by SanjalKatiyar** [comment]
/approve
/lgtm
will add "rehearsals-ack" after the tests...

**Pull request merged**